katas/content/single_qubit_gates/prepare_rotated_state/index.md

```diff
@@ -5,6 +5,6 @@
 
 **Goal:** Use a rotation gate to transform the qubit into state $\alpha|0\rangle -i\beta|1\rangle$.
 
-> You will probably need functions from the <a href="https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.math" target="_blank">Math</a> namespace, specifically <a href="https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.math.arctan2" target="_blank">ArcTan2</a>.
+> You will probably need functions from the `Microsoft.Quantum.Math` namespace, specifically <a href="https://learn.microsoft.com/en-us/qsharp/api/qsharp-lang/microsoft.quantum.math/arctan2" target="_blank">ArcTan2</a>.
```

Comment:
It's generally recommended to use links without en-us, so that other locales can use translated versions of pages.

```suggestion
> You will probably need functions from the `Microsoft.Quantum.Math` namespace, specifically <a href="https://learn.microsoft.com/qsharp/api/qsharp-lang/microsoft.quantum.math/arctan2" target="_blank">ArcTan2</a>.
```

Reply:
Good call, thanks!